ansible/configs/multi-region-tower/files/hosts_template.j2.back

```diff
@@ -0,0 +1,15 @@
+[all:vars]
+###########################################################################
+### Ansible Vars
+###########################################################################
+timeout=60
+ansible_become=yes
+ansible_user={{remote_user}}
+ansible_ssh_private_key_file="~/.ssh/{{guid}}key.pem"
+ansible_ssh_common_args="-o StrictHostKeyChecking=no"
+
+[nodes]
+## These are the frontends
+{% for host in groups['nodes']|d([]) %}
+node{{loop.index}}.{{chomped_zone_internal_dns}} ansible_ssh_host=frontend{{loop.index}}.{{subdomain_base}}
```

Comment:
i would prefer if we use `hostvars[host]` here instead of generating using `loop.index`